.github/workflows/build.yml

```diff
@@ -595,8 +595,69 @@ jobs:
           name: docs
           path: tmp/docs/
 
+  check-doc-links:
+    needs:
+      - build-docs
+
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
```

Comment:
The [GitHub Docs, `timeout-minutes`](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes) says:

> The maximum number of minutes to let a job run before GitHub automatically cancels it. Default: 360

> **Note:** The `GITHUB_TOKEN` expires when a job finishes or after a maximum of 24 hours.

Are you sure that 2 minutes is enough for the action to run succesfully? This timeout seems very low to me.

Reply:
It's okay, in my laptop runs for about 10 seconds